acestep/core/generation/handler/init_service_catalog.py

```
@@ -0,0 +1,73 @@
+            major, _ = torch.cuda.get_device_capability()
+            if major < 8:
+                logger.info(
+                    f"[is_flash_attention_available] GPU compute capability {major}.x < 8.0 "
+                    f"(pre-Ampere) — FlashAttention not supported, will use SDPA instead."
+                )
+                return False
+        except Exception:
+            return False
+
+        try:
+            import flash_attn  # noqa: F401
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)